src/pages/ProductDetails/ProductDetails.jsx

```diff
@@ -0,0 +1,57 @@
+import { useParams } from "react-router-dom";
+import {FaShippingFast,MdEventAvailable, FaShoppingCart, AiFillHeart, MdEventBusy} from "../../assets/icons";
+import { Button, Navbar, Ratings, TextBadgeSquare } from "../../Components/index";
+import { useFilteredData } from "../../Context/filterData.context"
+
```

Comment:
suggestion: give an absolute path 
refer this blog: https://dynamicprogrammer.hashnode.dev/absolute-path-in-react-js